specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/azureFirewall.json

```diff
@@ -304,6 +304,10 @@
       },
       "AzureFirewallPropertiesFormat":{  
          "properties":{  
+            "threatIntelMode":{
```

Comment:
@fanymanea could you move the property to the end of `properties` list?

Reply:
Doing it now...

Reply:
Done